recipe/meta.yaml

```diff
@@ -33,19 +33,19 @@ requirements:
         - xerces-c
         - libnetcdf
         - krb5
-        - libtiff
+        - libtiff 4*
         - libpng
         - openjpeg
         - curl
         - expat
         - zlib  # [not win]
-        - jpeg  # [not win]
+        - jpeg 9*  # [not win]
         - postgresql  # [linux]
         - jasper  # [not win]
         - giflib  # [not win]
         - json-c  # [not win]
         - freexl  # [not win]
-        - xz  # [not win]
+        - xz 5*  # [not win]
```

Comment:
I don't understand the `*` in pinning, but if that means any version `5` series it won't work for `libzma` because the available, and conflicting, `xz` versions are both from the `5.x` series (`5.0.5` and `5.2.2`).